flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/batch/sql/TableWithSQLITCase.scala

```diff
@@ -113,4 +114,26 @@ class TableWithSQLITCase(
     val results = result.toDataSet[Row].collect()
     TestBaseUtils.compareResultAsText(results.asJava, expected)
   }
+
+  @Test
+  def testUDTFWithPojoType(): Unit = {
```

Comment:
I found there was already a testPojoType case in DataSetUserDefinedFunctionITCase, so I extend it. And I extend testCrossJoin in DataStreamUserDefinedFunctionITCase, do you think it's OK?